scanner/src/main/kotlin/storages/PostgresStorage.kt

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2017-2019 HERE Europe B.V.
+ * Copyright (C) 2017-2021 HERE Europe B.V.
```

Comment:
Commit message: I vaguely remember @fviernau quoting some performance numbers in an issue or PR, and the speedup was quote impressive. Can you also quote these numbers as part of the commit message?

Reply:
I just started performance testing after creating the PR. It turns out the majority of the performance gain you mentioned was seen because the tests were run single-threaded. If the single requests are executed multi-threaded they perform a lot better and were actually faster than a bulk fetch in most cases. I have now tested combining both approaches to request chunks of scan results in parallel which improved performance by factor 2 in my tests. I still need to incorporate those changes into this PR.

Reply:
I have added some explanations about performance to the commit message.